### DISCUSSION TITLE
BIRT 4.9 New Features

### DISCUSSION THREAD

**Discussion by syc795** [eclipse-birt/birt#927] [Q&A] [answered]
Where to find the new features in birt 4.9?

**Comment by hvbtup** [comment]
There are no new features, just bug fixes, compatibility with Java 11, newer library versions.

**Reply by syc795** [reply]
I saw the [discussion](https://github.com/eclipse/birt/discussions/892) about using java 8 for birt 4.9. If birt 4.9 does not have new features, is it possible to compile using java 8 theoretically?
Can you provide a modified library version range, since I'm trying to compile birt using java 8. My product is currently using java 8.

**Reply by wimjongman** [reply]
Everything is possible but we will not do it. You will have to fork BIRT and do it yourself. 

It will be a big job. Much bigger than bringing your product to Java 11.

News in 4.9.0: https://github.com/eclipse/birt/milestone/1?closed=1

**Comment by wimjongman** [comment, marked as answer]
News in 4.9.0: https://github.com/eclipse/birt/milestone/1?closed=1
